transformer/transformer.go

```diff
@@ -235,6 +240,20 @@ func (t *Transformer) TransformSearchResponse(
 	return transformedData, nil
 }
 
+// TransformCountResponse transforms an elastic search 7.x response
+func (t *Transformer) TransformCountResponse(
+	ctx context.Context, responseData []byte) (int, error) {
+	var data struct {
+		Count int `json:"count"`
+	}
+
+	err := json.Unmarshal(responseData, &data)
```

Comment:
Combine line 250 with 251:

```suggestion
	if err := json.Unmarshal(responseData, &data); err != nil {
```